CMakeLists.txt

```diff
@@ -253,14 +274,20 @@ if (NOT EMSCRIPTEN)
     link_libraries(gcov)
   endif ()
 
+  set(OMR_GC OFF)
+  set(OMR_OMRSIG OFF)
+  set(OMR_JIT ON)
+  set(OMR_JITBUILDER ON)
+  add_subdirectory(third_party/omr)
+
   function(wabt_executable name)
     # ARGV contains all arguments; remove the first one, ${name}, so it's just
     # a list of sources.
     list(REMOVE_AT ARGV 0)
 
     add_executable(${name} ${ARGV})
     add_dependencies(everything ${name})
-    target_link_libraries(${name} libwabt)
+    target_link_libraries(${name} libwabt jitbuilder)
```

Comment:
You've added the library but you don't seem to have added JitBuilder's headers to the include path.

Reply:
The include paths are set by the JitBuilder CMake files as part of the build target's public interface. CMake will automatically add these to any target that links the JitBuilder library.

Reply:
Ah, alright. I rarely see that actually used, so I honestly forgot CMake could do that.